dll/win32/shell32/folders/CFSFolder.cpp

```diff
@@ -991,6 +991,17 @@ HRESULT WINAPI CFSFolder::EnumObjects(
     DWORD dwFlags,
     LPENUMIDLIST *ppEnumIDList)
 {
+    HRESULT hr;
+
+    if (FAILED(SHELL_FindAnyFile(m_sPathTarget)))
+    {
+        RETRY_DATA* retryData;
+        retryData = (RETRY_DATA*)SHAlloc(sizeof(RETRY_DATA));
```

Comment:
```suggestion
RETRY_DATA retryData;
```
and just pass `(LPARAM)&retryData` because RETRY_DATA is small enough and DialogBoxParamW is modal so this can live on the stack.